dataprofiler/profilers/column_profile_compilers.py

```diff
@@ -328,6 +328,35 @@ def profile(self):
             profile["data_label"] = profiler.data_label
             profile["statistics"].update(profiler.profile)
         return profile
+    
+    def diff(self, other, options=None):
+        """
+        Finds the difference between 2 compilers and returns the report
+
+        :param other: profile compiler finding the difference with this one.
+        :type other: ColumnDataLabelerCompiler
+        :return: difference of the profiles
+        :rtype: dict
+        """
+        # Call super for compiler instance check
+        diff_profile = super().diff(other, options)
+        diff_profile["statistics"] = dict()
+        
+        # Iterate through profile(s)
+        all_profiles = set(self._profiles.keys()) | set(other._profiles.keys())
+        for key in all_profiles:
+            if key in self._profiles and key in other._profiles:
+                diff_data_label = utils.find_diff_of_strings_and_bools(
+                    self._profiles[key].data_label, 
+                    other._profiles[key].data_label)
+                diff_profile["data_label"] = diff_data_label
+                diff = self._profiles[key].diff(other._profiles[key], options)
+                diff_profile["statistics"].update(diff)
+        
+        if diff_profile["statistics"] == {}:
+            diff_profile.pop("statistics")
```

Comment:
not needed change for approval, but if we were to match the style above, `if not diff_profile["statistics"]` as oppose tot he `== {}`